### PR TITLE
add hello-thirdparty sample

### PR DESCRIPTION
**Pull request by proppy**
/cc @claywilkinson @gguuss 

Should partially address #26 as it shows how to use ldFlags to link to a pre-built third-party lib (play game services)

### CODE REVIEW

File: native-playgameservices/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+1. [Download Android Studio](http://developer.android.com/sdk/index.html)
+1. unzip Play Game C++ SDK in the sample directory.
+  - it should create the following directory:
+        android-ndk/native-playgameservices/gpg-cpp-sdk
+1. Launch Android Studio.
+1. Open the sample directory.
+1. Open *File/Project Structure...*
+  - Click *Download* or *Select NDK location*.
+1. Click *Tools/Android/Sync Project with Gradle Files*.
+1. Set your `APP_ID` in `res/values/ids.xml` to the ID your project in the *Game services* of the Google Play console.
```
Comment (claywilkinson) [inline native-playgameservices/README.md]: Need to set the package id to match also.

File: native-playgameservices/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,40 @@
+apply plugin: 'com.android.model.application'
+
+model {
+    android {
+        compileSdkVersion = 23
+        buildToolsVersion = "23.0.0"
+
+        defaultConfig.with {
+            applicationId = "com.mycompany.android.codelab.proppy.lollygagger"
```
Comment (claywilkinson) [inline native-playgameservices/app/build.gradle]: this should be more sample like, no?

File: native-playgameservices/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- BEGIN_INCLUDE(manifest) -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="com.mycompany.android.codelab.proppy.lollygagger"
```
Comment (claywilkinson) [inline native-playgameservices/app/src/main/AndroidManifest.xml]: use a sample package id

File: native-playgameservices/app/src/main/jni/StateManager.h [hunk 4]
```diff
@@ -0,0 +1,44 @@
+#ifndef TEAPOT_JNI_STATE_MANAGER_H
```
Comment (claywilkinson) [inline native-playgameservices/app/src/main/jni/StateManager.h]: copyright?

File: native-playgameservices/app/build.gradle (hunk 2 above)
Comment (proppy) [inline native-playgameservices/app/build.gradle]: Yes, that was a leftover of my test.

Done.

File: native-playgameservices/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (proppy) [inline native-playgameservices/app/src/main/AndroidManifest.xml]: Done.

File: native-playgameservices/app/src/main/jni/StateManager.h (hunk 4 above)
Comment (proppy) [inline native-playgameservices/app/src/main/jni/StateManager.h]: Done.

File: native-playgameservices/README.md (hunk 1 above)
Comment (proppy) [inline native-playgameservices/README.md]: Done.

**Comment by proppy** [comment]
@gguuss I was under the impression we shouldn't update copyright year (unless we create new files, I guess we could consider that as new files even if they come from another repo).

**Comment by proppy** [comment]
PTAL

**Comment by gguuss** [comment]
LGTM

**Comment by proppy** [comment]
PTAL, made a bunch of unrelated change to make travis happy and renamed the package.

**Comment by gguuss** [comment]
LGTM.

**Pull request merged**